### ISSUE TITLE
replyQueue: false - messages do not get published

### ISSUE THREAD

**Issue by astanciu** [LeanKit-Labs/wascally#65]
Hi,

Not sure if I'm doing something wrong or if there's an issue elsewhere but I noticed that if I add the connection option `replyQueue: false` the publish() method does not actually publish anything. As soon as the option is removed, messages start going through.

```javascript
var rabbit = require('wascally');
var settings = {
    "connection": {
        "user": "admin",
        "pass": "password",
        "server": "localhost",
        "port": 5672,
        "vhost": "idio",
        "replyQueue": false  //No messages with this on. Comment out and messages get sent to queue
    },
    "exchanges": [
        { "name": "crud", "type": "direct", "durable": true, "persistent": true, "autoDelete": false }
    ],
    "queues": [
        { "name": "ops", "durable": true , "autoDelete": false} 
    ],
    "bindings": [
        { "exchange": "crud", "target": "ops", "keys": [] }
    ]
}


rabbit.configure(settings)


function sendMessage(obj) {
    rabbit.publish('crud', {
        type: 'idio.message',
        routingKey: '',
        body: obj
    });
    console.log('Sent: ' + obj.id)
}


function send(total) {
	console.time('mq')
    for (var i = 1; i <= total; i++) {
        var obj = {
            id: i,
            name: 'Hello ' + i,
            total: total
        }
        sendMessage(obj)
    }
}

send(10)

```

**Comment by juliendangers** [comment]
yes, we observe the same thing. There is a silent fail when you set replyQueue to false. 

Actually you have to set a replyQueue (even if you don't use it), or you can't publish anything :/

**Comment by arobson** [comment]
@astanciu @juliendangers - this should be fixed in 0.2.5 now. Please let me know if you continue to notice issues related to this feature.

**Comment by regaw-leinad** [comment]
This issue seems to have come back up. I am running version 0.2.10, using the exact same sample code as in the OP.

More specifically, in my testing with this code, the messages are not delivered to the queue even if I declare `replyQueue: true`. It only delivers the messages to the queue when `replyQueue` is not defined at all in the `settings.connection` hash.
